string text_1

int32 value_1

float64 value_2

uint8[] vector_1
